src/main/java/ch/uzh/ifi/hase/soprafs22/game/GameDelta.java

```diff
@@ -24,15 +24,15 @@ public GameDelta(MoveCommand moveCommand,
     }
 
     public GameDelta(MoveCommand moveCommand, Map<Position, Integer> unitHealths, TurnInfo turnInfo, GameOverInfo gameOverInfo) {
-        this(moveCommand, unitHealths, turnInfo, gameOverInfo, /*surrenderInfo=*/ null);
```

Comment:
Can you leave the comments for null variables? They explain what the null actually is, which is much more readable. I added them on purpose.

Reply:
<img width="949" alt="Bildschirmfoto 2022-05-28 um 20 43 49" src="https://user-images.githubusercontent.com/91155454/170838823-53c22f46-f526-471a-9831-d76f6f0a4745.png">

IntelliJ does this job for me.

Reply:
thank you. :smiling_face_with_tear:

Reply:
yes, but github does not

Reply:
understood and comments reapplied ;)

Reply:
:+1: